HIF.md

```diff
@@ -474,154 +474,108 @@ of these commands for a more detailed explanation.
 
- - `uint8_t ffn_bc_clock_drift`  
-    (unused)
+ - `struct chan_seq`  
+   [Channel sequence][chan-seq] description.
 
- - `uint8_t ffn_bc_timing accuracy`  
-    (unused)
+<!-- TODO: document parent BS-IE following -->
 
 ### `0x32 SET_FHSS_LFN_BC`
 
- - `uint16_t flags`  
-    A bitfield:
-     - `0x0030: FHSS_CHAN_FUNC_MASK`:
-        - `0: FHSS_CHAN_FUNC_FIXED`: Use Fixed Channel
-        - `1: FHSS_CHAN_FUNC_TR51`: Use TR51
-        - `2: FHSS_CHAN_FUNC_DH1`: Use DH1
-        - `3: RESERVED`: (FHSS_CHAN_FUNC_AUTO)
+Configure broadcast schedule for transmission to LFN.
 
-Only present if `FHSS_CHAN_FUNC_FIXED`:
-
- - `uint16_t chan_nr`
-    Fixed channel value.
-
-Only present if `FHSS_CHAN_FUNC_DH1`:
-
- - `uint8_t chan_mask_len`  
-    Length of the next field. Maximum value is 64.
-
- - `uint8_t chan_mask[]`  
-    Bitmap of masked channels.
-
-Only present if `FHSS_CHAN_FUNC_TR51`:
+ - `uint16_t interval`  
+    LFN broadcast interval in milliseconds (from LBS-IE).
 
-  TBD
+ - `uint16_t bsi`  
+    LFN broadcast Schedule ID (from LBS-IE).
 
-Always present:
+ - `struct chan_seq`  
+   [Channel sequence][chan-seq] description.
 
- - `uint32_t lfn_bc_interval_ms`  
+> [!WARNING]
+> The current RCP implementation uses the same set of parameters for FFN and
+> LFN brodcast channel sequences. Thus, [`SET_FHSS_FFN_BC`][bc] and
+> [`SET_FHSS_LFN_BC`][bc-lfn] override each other's channel sequence. The other
+> parameters are distinct.
 
- - `uint16_t lfn_bc_broadcast_schedule_id`  
-
- - `uint8_t lfn_bc_sync_period`  
+> [!NOTE]
+> The host is responsible for periodically sending LFN Time Sync frames in
+> order to maintain broadcast timing in LFN children.
 
 ### `0x33 SET_FHSS_ASYNC`
 
- - `uint16_t flags`  
-    A bitfield:
-     - `0x0030: FHSS_CHAN_FUNC_MASK`:
-        - `0: FHSS_CHAN_FUNC_FIXED`: Use Fixed Channel
-        - `1: FHSS_CHAN_FUNC_TR51`: Use TR51
-        - `2: FHSS_CHAN_FUNC_DH1`: Use DH1
-        - `3: RESERVED`: (`FHSS_CHAN_FUNC_AUTO`)
-
-Only present if `FHSS_CHAN_FUNC_FIXED`:
-
- - `uint16_t chan_nr`  
-    Fixed channel value.
-
-Only present if `FHSS_CHAN_FUNC_DH1`:
-
- - `uint8_t chan_mask_len`  
-    Length of the next field. Maximum value is 64.
+Configure asynchronous transmissions for network discovery.
 
- - `uint8_t chan_mask[]`  
-    Bitmap of masked channels.
+ - `uint32_t tx_duration`  
+    Maximum number of milliseconds the RCP is allowed to stay in continuous TX
+    for an async transmission. If that duration is exceeded, the async
+    transmission is split into chunks of that duration until all channels have
+    been used. By default, the value is set to `0xffffffff`, which disables this
+    feature.
 
-Only present if `FHSS_CHAN_FUNC_TR51`:
+  - `uint8_t chan_mask_len`  
+     Number of bytes in the following channel mask.
 
-  TBD
+  - `uint8_t chan_mask[]`  
+     Bitmask of channels to use for transmission. See the same field in
+     ["Channel sequence"][chan-seq] for more details.
```

Comment:
`SET_FHSS_ASYNC` do not use a `struct chan_seq`?

Reply:
No, there's no such thing as a channel function for async.